Source/ASCollectionNode.mm

```diff
@@ -34,6 +34,20 @@
 @interface _ASCollectionPendingState : NSObject {
 @public
   std::vector<std::vector<ASRangeTuningParameters>> _tuningParameters;
+
+  struct {
+    ASLayoutRangeMode rangeMode;
+    ASCellLayoutMode cellLayoutMode;
+    unsigned int allowsSelection:1; // default is YES
+    unsigned int allowsMultipleSelection:1; // default is NO
+    unsigned int inverted:1; //default is NO
+    unsigned int alwaysBounceVertical:1;
+    unsigned int alwaysBounceHorizontal:1;
+    unsigned int animatesContentOffset:1;
+    unsigned int showsVerticalScrollIndicator:1;
+    unsigned int showsHorizontalScrollIndicator:1;
+    unsigned int pagingEnabled:1;
+  } _fields;
```

Comment:
Maybe flags instead of fields?

Reply:
There are a couple of enums in there too, so I didn't want to use the name `_flags`. I'll see what happens to the size if I move them out of the struct but keep them next to this bitfield struct.

Reply:
ok. new diff to come.